### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by HarshnaNHaswani**
image
image
image
image

validation:

image
image

### CODE REVIEW

File: src/pages/auth/Login.jsx [hunk 1]
```diff
@@ -1,5 +1,164 @@
...
+  const { isLoggedIn, setIsLoggedIn } = useAuth();
+  const navigate = useNavigate();
+  const loginHandler = async (event) => {
+    event.preventDefault();
+    try {
+      setLoadState(true);
+      setError(null);
+      const response = await axios.post("/api/auth/login", {
+        email: loginData.email,
+        password: loginData.password,
```
Comment (git-0r) [inline src/pages/auth/Login.jsx:30]: suggestion: you can extract this part into a separate file or custom hook.

File: src/pages/auth/Login.jsx [hunk 2]
```diff
@@ -1,5 +1,164 @@
...
+      }
+    }
+  };
+
+  // useEffect(() => {
+  //   if (isLoggedIn) {
+  //     location?.state?.from?.pathname
+  //     ? navigate(location.state.from.pathname, { replace: true })
+  //     : navigate("/");
+  //   }
```
Comment (git-0r) [inline src/pages/auth/Login.jsx:76]: remove the comments

**Review by sashaboi — APPROVED** [review]
The UI is just FAB!
I am looking forward to seeing you add p2 features later down the road!

File: src/pages/auth/Login.jsx [hunk 3]
```diff
@@ -1,5 +1,164 @@
...
+            loginData.email ? "status-success-outline" : "status-plain-outline"
+          }`}
+        >
+          <input
+            type="email"
+            className="form-input"
+            id="email"
+            placeholder="email address"
+            aria-label="email"
+            autoComplete="email"
```
Comment (git-0r) [inline src/pages/auth/Login.jsx:96]: all these inputs will rerender on change, you can make smaller components

**Review by git-0r — APPROVED** [review]

**Comment by HarshnaNHaswani** [comment]
Thanks for your time and suggestions, I'll make the necessary changes @sashaboi  @git-0r

**Pull request merged**